drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/handlers/ScenarioSimulationGridPanelClickHandler.java

```diff
@@ -113,14 +116,26 @@ public void setEventBus(EventBus eventBus) {
 
     @Override
     public void onClick(ClickEvent event) {
-        hideMenus();
-        scenarioGrid.clearSelections();
-        if (manageLeftClick(event)) {
-            event.preventDefault();
-            event.stopPropagation();
-        } else {
-            eventBus.fireEvent(new DisableRightPanelEvent());
-        }
+        clickReceived.getAndIncrement();
+        final int canvasX = getRelativeX(event);
+        final int canvasY = getRelativeY(event);
+        final boolean isShiftKeyDown = event.getNativeEvent().getShiftKey();
+        final boolean isControlKeyDown = event.getNativeEvent().getCtrlKey();
+        Timer t = new Timer() {
+            @Override
+            public void run() {
+                if (clickReceived.get() ==1) {
+                    hideMenus();
+                    scenarioGrid.clearSelections();
+                    if (manageLeftClick(canvasX, canvasY, isShiftKeyDown, isControlKeyDown)) {
+                    } else {
+                        eventBus.fireEvent(new DisableRightPanelEvent());
+                    }
+                }
+                clickReceived.set(0);
+            }
+        };
+        t.schedule(300);
```

Comment:
Please add a comment to explain the reason of this (= manage single click and double click on the same element)